website/redirects.js

```diff
@@ -30,67 +30,67 @@ module.exports = [
   {
     source: "/dl/firezone-client-gui-linux/latest/x86_64",
     destination:
-      // mark:automatic-version
-      "https://www.github.com/firezone/firezone/releases/download/1.0.5/firezone-client-gui-linux_1.0.5_x86_64.deb",
+      // mark:current-gui-version
+      "https://www.github.com/firezone/firezone/releases/download/gui-client-1.0.5/firezone-client-gui-linux_1.0.5_x86_64.deb",
     permanent: false,
   },
   {
     source: "/dl/firezone-client-gui-linux/latest/aarch64",
     destination:
-      // mark:automatic-version
-      "https://www.github.com/firezone/firezone/releases/download/1.0.5/firezone-client-gui-linux_1.0.5_aarch64.deb",
+      // mark:current-gui-version
+      "https://www.github.com/firezone/firezone/releases/download/gui-client-1.0.5/firezone-client-gui-linux_1.0.5_aarch64.deb",
     permanent: false,
   },
   {
     source: "/dl/firezone-client-headless-linux/latest/x86_64",
     destination:
-      // mark:automatic-version
-      "https://www.github.com/firezone/firezone/releases/download/1.0.5/firezone-client-headless-linux_1.0.5_x86_64",
+      // mark:current-headless-version
+      "https://www.github.com/firezone/firezone/releases/download/headless-client-1.0.5/firezone-client-headless-linux_1.0.5_x86_64",
     permanent: false,
   },
   {
     source: "/dl/firezone-client-headless-linux/latest/aarch64",
     destination:
-      // mark:automatic-version
-      "https://www.github.com/firezone/firezone/releases/download/1.0.5/firezone-client-headless-linux_1.0.5_aarch64",
+      // mark:current-headless-version
+      "https://www.github.com/firezone/firezone/releases/download/headless-client-1.0.5/firezone-client-headless-linux_1.0.5_aarch64",
     permanent: false,
   },
   {
     source: "/dl/firezone-client-headless-linux/latest/armv7",
     destination:
-      // mark:automatic-version
-      "https://www.github.com/firezone/firezone/releases/download/1.0.5/firezone-client-headless-linux_1.0.5_armv7",
+      // mark:current-headless-version
+      "https://www.github.com/firezone/firezone/releases/download/headless-client-1.0.5/firezone-client-headless-linux_1.0.5_armv7",
     permanent: false,
   },
   // versioned
   {
     source: "/dl/firezone-client-gui-linux/:version/x86_64",
     destination:
-      "https://www.github.com/firezone/firezone/releases/download/:version/firezone-client-gui-linux_:version_x86_64.deb",
+      "https://www.github.com/firezone/firezone/releases/download/gui-client-:version/firezone-client-gui-linux_:version_x86_64.deb",
     permanent: false,
   },
   {
     source: "/dl/firezone-client-gui-linux/:version/aarch64",
     destination:
-      "https://www.github.com/firezone/firezone/releases/download/:version/firezone-client-gui-linux_:version_aarch64.deb",
+      "https://www.github.com/firezone/firezone/releases/download/gui-client-:version/firezone-client-gui-linux_:version_aarch64.deb",
```

Comment:
Should we just call it a `client` so we have `client` and `headless_client`? Like it's done for `chrome` and `headless_chrome`.

Reply:
Hmm but then we will have `client` Docker image which is the headless client, and then `client` deb which is the GUI client.

Were you talking about renaming the file name, permalink, or the release name?

Reply:
I was talking about the file name, not sure what the release name affects. Permalink would be nice too I guess.

I would rename `client` container also to `headless_claint` to ensure naming is consistent.

But this is not critical, we can go with whatever this PR implements this too.

Reply:
Yeah, it's a little verbose, but it's what @ReactorScram and I settled on due to the complexity of managing it all. There's a lot of renaming and moving stuff around for that.